### PR TITLE
Update tutorial-windows10.md

### PR DESCRIPTION
**Pull request by Mhudson97**
### Summarize the change in the pull request title

Admins need to understand that this policy that is created is not shown in the Client settings but via the Azure settings plus this sets a policy on the machine.  Any Group policy set for Data Collection will cause a fight with Desktop Analytics.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger18** [comment]
@Mhudson97 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/desktop-analytics/tutorial-windows10.md [hunk 1]
```diff
@@ -219,6 +219,13 @@ Install the Configuration Manager version 1902 update rollup (4500571) to suppor
 
 Configuration Manager creates a settings policy to configure devices in the Target Collection. This policy includes the diagnostic data settings to enable devices to send data to Microsoft. By default, clients update policy every hour. After receiving the new settings, it can be several hours more before the data is available in Desktop Analytics.
 
+>[!Note]  
+>To confirm machines settings by Desktop Analytics refer to the following policy key:
```
Comment (aczechowski) [inline sccm/desktop-analytics/tutorial-windows10.md]: Can we change this to instead just point to the following section?

```suggestion
> For more information on these settings, see [Windows settings](/sccm/desktop-analytics/enroll-devices#windows-settings).
```

(and then nix the additional lines of the note)

Comment (Mhudson97) [reply sccm/desktop-analytics/tutorial-windows10.md]: Yes, agreed....

**Comment by PRMerger16** [comment]
@aczechowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
#sign-off

**Pull request merged**